src/test/tpc/tpch_test.cpp

```diff
@@ -60,7 +60,8 @@ TEST_P(TPCHTest, Test) {
   SCOPED_TRACE("TPC-H " + std::to_string(tpch_idx) + (use_jit ? " with JIT" : " without JIT") + " and " +
                (use_prepared_statements ? " with prepared statements" : " without prepared statements"));
 
-  auto query_generator = TPCHQueryGenerator{use_prepared_statements};
+  // The scale factor passed to the query generator will be ignored as we only use deterministic queries
+  auto query_generator = TPCHQueryGenerator{use_prepared_statements, 1.0f};
```

Comment:
Should we add a default parameter for the scale factor then, so that if we don't care about it we don't have to pass a random float?

Reply:
I wouldn't say so. If you actually generate queries, you must set it - otherwise your queries would be invalid. It is just that in this case, we only care about the prepare statements where this does not make a difference.

Reply:
Yeah that makes sense.